graphene_django/range_types.py

```diff
@@ -0,0 +1,29 @@
+from graphene import ObjectType
+from graphene import (
+    Float,
+    Int,
+    DateTime,
+    Date
+)
+
+class RangeResolver:
+    def resolve_lower(parent, info):
+        return parent.lower
+    def resolve_upper(parent, info):
+        return parent.upper
+
+class DateTimeRangeType(RangeResolver, ObjectType):
```

Comment:
According to http://initd.org/psycopg/docs/extras.html#adapt-range `bounds` and `empty` fields are missing.

Reply:
bounds is not accessible. you said missing is referring to mutation?

Reply:
@jkimbo here is attributes:
```
Object attributes:

isempty
True if the range is empty.

lower
The lower bound of the range. None if empty or unbound.

upper
The upper bound of the range. None if empty or unbound.

lower_inc
True if the lower bound is included in the range.

upper_inc
True if the upper bound is included in the range.

lower_inf
True if the range doesn’t have a lower bound.

upper_inf
True if the range doesn’t have an upper bound.
```

I see original method either not implement it, so I didn't too.